### DISCUSSION TITLE
Discord vs Slack

### DISCUSSION THREAD

**Discussion by zcanfes** [bounswe/2021SpringGroup9#17] [General]
As we discussed in this week's meeting, we thought it would be a good idea to get more information on Slack to decide which platform will be better for us. 

I have gathered information accordingly and would appreciate if you can tell me your opinions on this. 

The **pros** and **cons** of both platforms:

![Caption.](https://user-images.githubusercontent.com/56366573/112733947-e1e6f800-8f53-11eb-81a8-8f1b9ab3bc87.png)https://www.chanty.com/blog/discord-vs-slack/

### Pros

1. User interface is more convenient for work use
2. Limit for file sharing is 1 GB
3. There are conversation threads which keep the chat organized and easy to access
4. There are different channels for different topics

### Cons

1. Video conversations is limited among 2 people in the free plan
2. Screen sharing is not permitted in the free plan (which we will use often probably)
3. Does not have channels for voice chat (we can need more voice channels in the future to differentiate between meetings, chats etc.)
4. It is required to download the desktop or mobile apps to use Slack
5. Limited chat history in the free plan
6. Storage is limited with 5 GB in the free plan

### Pros 

1. Video conversations are available and not limited 
2. Screen sharing is permitted 
3. We can have different voice channels and text channels
4. We can use the web version and desktop apps 
5. Storage is unlimited
6. Message history is unlimited

### Cons

1. File upload is limited with 8 MB which is very low compared to Slack
2. There are no conversation threads 

**I tried to gather the most important features for us.**

From my point of view, since we are going to use free plan, Discord will provide a better medium for our project. 
We can add more text channels to keep Discord organized. 

We can vote under this discussion to decide which platform to use. 

If you have other ideas we can discuss them. 

#### References

- https://www.chanty.com/blog/discord-vs-slack/

- https://www.troopmessenger.com/blogs/discord-vs-slack-which-one-is-better-for-businesses

**Comment by zcanfes** [comment]
If you think we should use Discord give a reaction with "+1", else if you think we should use Slack give "-1" to this comment,

**Comment by niyaziulke** [comment]
Thanks for the detailed analysis. Screen sharing really makes a difference.
